inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt

```diff
@@ -0,0 +1,22 @@
+defusedxml>=0.5.0
+scipy==1.2.1
```

Comment:
@AlexKoff88 @MaximProshin could you please suggest another scipy version (it does not support python3.8+ and can not be installed on windows with such python version)

Reply:
@eaidova, let's put it this way. What version you need for AC. We could adjust POT: update to the recent version of AC and verify that it works.

Reply:
we do not pin version explicitly, but currently in testing used 1.5.4, is it ok for you?

Reply:
we need to verify that.

Reply:
Updated: scipy==1.5.4